### ISSUE TITLE
quickbar reload doesnt work anymore/hangs

### ISSUE THREAD

**Issue by Mariusthvdb** [home-assistant/frontend#11714]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

filed this https://github.com/home-assistant/core/issues/66700 because it is about reloading of core options, but probably a frontend thing too?

Reloading via quickbar is (at least partially) disfunctional at the moment on dev20220216

### Describe the behavior you expected

reload doesnt hang, and actually reloads

### Steps to reproduce the issue

1. click C key to load quickbar
2. groups
3. reload.
...

most of the time I can force reload the view and see the reload was done after all, sometimes the only way is to reload via the server controls page. Not sure when exactly.

### What version of Home Assistant Core has the issue?

dev20220216

### What was the last working version of Home Assistant Core?

version before

### In which browser are you experiencing the issue with?

Safari/Chrome/Firefox on Mac

### Javascript errors shown in your browser console/inspector
console:
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'action')
```

homeassistant.log:
```
2022-02-17 10:18:01 ERROR (MainThread) [frontend.js.latest.202202140] :0:0 Script error.
```

### Additional information
<img width="632" alt="Schermafbeelding 2022-02-17 om 10 12 08" src="https://user-images.githubusercontent.com/33354141/154443274-acc7054b-c2f3-4f63-819b-39ccf24de155.png">

**Comment by dklemm** [comment]
More info : the issue occurs when when clicking the button but not the text.

**Comment by dklemm** [comment]
Already fixed #11900.

**Comment by Mariusthvdb** [comment]
believe all to be fixed (using 2022.4.0.dev20220323)

- no more console error
- no more frontend error
- no more hangs, and all reloads work as expected.
closing, that for fixing this!
